tests/Fixtures/http/200-delivery-options

```diff
@@ -1,7 +1,7 @@
 HTTP/1.1 200 OK
```

Comment:
Voor deze file missen, zoals besproken, nog een aantal keys. Ook weet ik niet zeker of de headers kloppen?

Reply:
Dit is een dump van een request die ik met een oude versie gedaan heb. Het zal vast niet helemaal up to date zijn met de nieuwste versie. Maar dat is ook niet nodig voor de test waar deze gebruikt wordt.

Reply:
Eens, was vooral ook als check bedoeld 🙂